Security/Voter/HandleCommandVoter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace League\Tactician\Bundle\Security\Voter;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\AccessDecisionManagerInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+/**
+ * Voter for security checks on handling commands.
+ *
+ * @author Ron Rademaker
+ */
+class HandleCommandVoter extends Voter
+{
+    /**
+     * The decision manager.
+     *
+     * @var AccessDecisionManagerInterface
+     */
+    private $decisionManager;
+
+    /**
+     * Command - Require role mapping
+     *
+     * @var array
+     */
+    private $commandRoleMapping;
+
+    /**
+     * Create a new HandleCommandVoter.
+     *
+     * @param AccessDecisionManagerInterface $decisionManager
+     * @param array $commandRoleMapping
+     * @param string $defaultRole
+     */
+    public function __construct(AccessDecisionManagerInterface $decisionManager, array $commandRoleMapping = [])
+    {
+        $this->decisionManager = $decisionManager;
+        $this->commandRoleMapping = $commandRoleMapping;
+    }
+
+    /**
+     * The voter supports checking handle commands
+     *
+     * @param string $attribute
+     * @param object $subject
+     * @return bool
+     */
+    protected function supports($attribute, $subject)
+    {
+        return $attribute === 'handle' && is_object($subject);
```

Comment:
Since the voter is on by default, we should probably document the 'handle' key and/or make the voter only enabled when there's config here?